.travis.yml

```diff
@@ -10,9 +10,9 @@ cache:
     - $HOME/.gradle/wrapper/
 
 # required for oraclejdk9
-sudo: required
+sudo: false
 dist: trusty
-group: edge
+group: beta
```

Comment:
Does it work this time? Since we have the other PRs still open for release/2.x. Also do we want to introduce the changes there too?